marketplace/emoji-tablet/emojiTablet.js

```diff
@@ -0,0 +1,72 @@
+///
+/// emojiTablet.js
+/// A tablet app for sending emojis to other users
+/// 
+/// Author: Elisa Lupin-Jimenez
+/// Copyright High Fidelity 2017
+///
+/// Licensed under the Apache 2.0 License
+/// See accompanying license file or http://apache.org/
+///
+/// All assets are under CC Attribution Non-Commerical
+/// http://creativecommons.org/licenses/
+///
+
+var lib = Script.require("https://hifi-content.s3.amazonaws.com/elisalj/emoji_scripts/emojiLib.js?" + Date.now());
+
+(function() {
+
+    var APP_NAME = "EMOJIS";
+    var APP_URL = "https://hifi-content.s3.amazonaws.com/elisalj/emoji_scripts/emojiTabletUI.html?" + Date.now();
```

Comment:
We should have these URLs relative rather than absolute

Reply:
I've tried making htmls relative, but they don't load properly, I had to include the full path, but maybe I'm not setting them correctly